spec/support/jasmine-browser.json

```diff
@@ -0,0 +1,18 @@
+ {
+   "srcDir": "public/assets",
+   "srcFiles": [
+     "govuk-admin-template-*.js",
```

Comment:
```suggestion
```
We don't need this line because it wasn't a file used in the past test configuration: https://github.com/alphagov/collections-publisher/blob/301003f7bbfe437e7dda5413419c4dcd050b3b11/spec/javascripts/support/jasmine.yml#L13-L14

Reply:
Thank you @kevindew